ACMILabs/spotlights
# #96 Add scroll arrows and update call to action

Resolves #96

As a UX design I would like to indicate to visitors that it is possible to select other spotlight videos from the same screen.

### Acceptance Criteria
- [x] Update call to action text to `TAP LENS ON READER` and animate collect arrow
- [x] Add scroll arrows

### Relevant design files
* None

### Testing instructions
1. Set your config to XOS Staging (see below)
1. Start your environment: `cd development` and `docker-compose up --build`
1. Load spotlights in your browser: http://localhost:8081
1. Note new call to action text looks correct
1. Note lens taps look correct
1. Note the right and left arrows function as expected
1. Test this for real in the Demo Lab: https://dashboard.balena-cloud.com/devices/29ade855383c7bafe7d0e5c7a3317764

```python
XOS_API_ENDPOINT=https://xos-stg.acmi.net.au/api/
AUTH_TOKEN=<SNIP>
# Spotlights
XOS_PLAYLIST_ID=156
```

### DoD
For requester to complete:
